### DISCUSSION TITLE
enable codeQL

### DISCUSSION THREAD

**Discussion by jukzi** [eclipse-platform/.github#141] [Ideas]
CodeQL https://codeql.github.com/ can hint to problems in sourcecode

Unlike SonarCube it tries to produce warnings only for serious flaws. 

Which are only a few  for platform.ui as an example:
https://github.com/jukzi/eclipse.platform.ui/security/code-scanning

**Comment by HeikoKlare** [comment]
Looks interesting. I cannot see the scan for your platform.ui clone, @jukzi. Can you set it public or post the results? Thanks!

**Reply by jukzi** [reply]
(was scrrenshot)

**Reply by jukzi** [reply]
i don't find a button to make it public. however it's even a good feature, that such reports can only be seen by commiters.

**Reply by HeikoKlare** [reply]
Valid point 🙂 Thanks for sharing the results!

**Comment by HannesWell** [comment]
Sounds good to me.

**Comment by jukzi** [comment]
when logged in, can you see 
https://github.com/eclipse-platform/eclipse.platform.ui/security/code-scanning?query=pr%3A986
?

**Reply by merks** [reply]
I can see that page.

**Reply by jukzi** [reply]
I enabled codeQL for platform.ui. You can see the results in the security tab:
![image](https://github.com/eclipse-platform/.github/assets/51790620/3b79a7f6-58bb-4bcd-a408-70a31400e7c2)
https://github.com/eclipse-platform/eclipse.platform.ui/security/code-scanning


@carstenartur
i see you tried similar on https://github.com/eclipse-platform/eclipse.platform.ui/pull/175
if you have any improvement please let us know.
